### PR TITLE
update doc per feedback from the Mizar team

### PR DESCRIPTION
**Pull request by chenqianfzh**
This PR makes miscellaneous updates per feedback from the Mizar team on Mizar-Arktos integration.

1. Update docs/instructions.
2. Find tune the Arktos-up.sh to support a default network template for mizar.

### Veriification
Verified in the my dev box, when env var ARKTOS_NETWORK_TEMPLATE=mizar, the default network object created under the system tenant when the Arktos cluster is up is:

```bash
NAME      TYPE    VPC                      PHASE   DNS
default   mizar   system-default-network    
```

### CODE REVIEW

File: hack/runtime/default_mizar_network.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "metadata": {
+        "name": "default",
+        "finalizers": ["arktos.futurewei.com/network"]
+    },
+    "spec": {
+        "type": "vpc",
```
Comment (h-w-chen) [inline hack/runtime/default_mizar_network.json:7]: should be "mizar" in this case

**Review by h-w-chen — COMMENTED** [review]
LGTM. Intertionally left unapproved to seek approval from mizar team.

File: docs/design-proposals/multi-tenancy/multi-tenancy-network.md [hunk 2]
```diff
@@ -259,7 +259,7 @@ Network object specifies service IPAM as *external*:
     spec:
       type: flat
```
Comment (h-w-chen) [inline docs/design-proposals/multi-tenancy/multi-tenancy-network.md:260]: please help to change type to "vpc" in this sample; "flat" has very little sense to have external ipam - this was defect in the original doc.

File: hack/runtime/default_mizar_network.json (hunk 1 above)
Comment (chenqianfzh) [reply hack/runtime/default_mizar_network.json:7]: will update

File: docs/setup-guide/arktos-enforces-netowrk-feature.md [hunk 3]
```diff
@@ -21,25 +21,55 @@ If this machine will be used as the master of a multi-node cluster, please set a
...
 ./cluster/kubectl.sh get net
 NAME      TYPE   VPC   PHASE   DNS
 default   flat
 ```
 
-4. Start the arktos-network-controller
+The output will be as follows if you set ARKTOS_NETWORK_TEMPLATE=mizar.
+```bash
+NAME      TYPE   VPC                      PHASE   DNS
+default   vpc    system-default-network
```
[the remark this answers is not in the document]
Comment (chenqianfzh) [reply docs/setup-guide/arktos-enforces-netowrk-feature.md:69]: Will be mizar after the change in the following json file as suggested.

File: docs/design-proposals/multi-tenancy/multi-tenancy-network.md (hunk 2 above)
Comment (chenqianfzh) [reply docs/design-proposals/multi-tenancy/multi-tenancy-network.md:260]: will update.

**Review by phudtran — APPROVED** [review]
Overall looks good to me!
Just some minor nitpicks to fix typos.
Also the name of the file "arktos-enforces-netowrk-feature.md" -> "arktos-enforces-network-feature.md"

**Review by zmn223 — APPROVED** [review]

**Pull request merged**